### ISSUE TITLE
Property logLevel is deprecated, but required in type 'LocalTrackOptions'

### ISSUE THREAD

**Issue by Talb2005** [twilio/twilio-video.js#1826]
When creating a new LocalVideoTrack / LocalAudioTrack / LocalDataTrack, providing **LocalTrackOptions** with {name: 'track-name'}  causes a Typescript error: 

```
Argument of type '{ name: string; }' is not assignable to parameter of type 'LocalTrackOptions'.
  Property 'logLevel' is missing in type '{ name: string; }' but required in type 'LocalTrackOptions'.ts(2345)
```

Since `logLevel` is **deprecated**, it should at least be marked as optional, but is currently marked as a required property of LocalTrackOptions.

 - [X] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [X] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [X] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [X] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

```
function toLocalVideoTrack(track: MediaStreamTrack): LocalVideoTrack {
            return new LocalVideoTrack(track, { name: 'camera' });
        }
```

**Expected behavior:**

No type error should occur when providing LocalTrackOptions without logLevel property.

**Actual behavior:**

Providing LocalTrackOptions without logLevel property causes a Typescript error.

TODO

**Software versions:**

 - [X] Browser(s): Not Applicable
 - [X] Operating System: macOS Monterey 12.4 
 - [X] twilio-video.js: 2.22.0
 - [X] Third-party libraries (e.g., Angular, React, etc.): Angular 14

**Comment by manjeshbhargav** [comment]
@Talb2005 ,

Thanks for reporting this issue. We are taking a look at it, and will get back to you with updates soon.

**Comment by Talb2005** [comment]
@manjeshbhargav @charliesantos I've created a PR with a fix. 
I think you can just merge it and then close this issue..

**Comment by PikaJoyce** [comment]
Hi @Talb2005,

This issue has now been fixed on the [latest version 2.24.1](https://github.com/twilio/twilio-video.js/releases/tag/2.24.1). Please let us know if you have any other issues!

Best,
Joyce
